### ISSUE TITLE
Much Lower performance on ResNet101, scripts for reproducing needed on ResNet101

### ISSUE THREAD

**Issue by dywu98** [lxtGH/DecoupleSegNets#39]
I followed your instructions to train a ResNet101-DeeplabV3+-decouple just like your provided script to train the ResNet50-DeeplabV3+-decouple. Despite I got a decent results for the first stage to train the base modle, I only got 81.8 after I trained the ResNet101-DeeplabV3+-decouple for the second stage，**which is 1.7 miou lower than the claimed performance in your ECCV passage.**  I've tried many different settings like changing the learning rate of the second stage, changing the weight of the joint_edgeseg_loss, but 81.8 (ms+flip inference) is the best results I can get using the provided code.

So could you please, or anyone who have reproduced the claimed results on ResNet101, provide the scripts for reproducing your claimed ResNet101 results in the passage?
Although you have provided the trained modle, I am pretty sure that there gotta be someone like me who are desperate to reproducing this amazing work on ResNet101.

**Comment by focusOnxx** [comment]
你好，我在复现这篇代码的时候发现Lbody的loss是30多，比其他loss大太多，请问这正常吗

**Comment by focusOnxx** [comment]
Hello, when I reproduced this code, I found that the loss of Lbody is over 30, which is much larger than other losses. Is this normal?
